### ISSUE TITLE
npm start failed: "Not a valid Eletron app ... /node_modules/sqlite3/lib/binding/node-v47-darwin-x64/node_sqlite3.node"

### ISSUE THREAD

**Issue by qbig** [brave/browser-laptop#221]
![screen shot 2016-01-21 at 1 59 02 pm](https://cloud.githubusercontent.com/assets/5563225/12472903/ed032794-c047-11e5-90d4-9914a2c12bd5.png)

**Comment by vmx** [comment]
On Debian I had  directory called `...v46..` instead of `...v47...`, symlinking it worked for me.

**Comment by vmx** [comment]
Seems to be duplicate of #219.

**Comment by qbig** [comment]
I was using node 5.0. would try with 5.5 and update here.

**Comment by qbig** [comment]
Node 5.5, npm 3.5.3

```
167053 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/appdmg/package.json'
167054 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/ampersand-version/package.json'
167055 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/archiver/package.json'
167056 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/ampersand-sync/package.json'
167057 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-code-frame/package.json'
167058 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/argparse/package.json'
167059 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-generator/package.json'
167060 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helper-call-delegate/package.json'
167061 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helper-define-map/package.json'
167062 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helper-function-name/package.json'
167063 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helper-builder-react-jsx/package.json'
167064 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helper-regex/package.json'
167065 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helper-replace-supers/package.json'
167066 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helpers/package.json'
167067 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helper-hoist-variables/package.json'
167068 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helper-get-function-arity/package.json'
167069 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-helper-optimise-call-expression/package.json'
167070 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-messages/package.json'
167071 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-syntax-jsx/package.json'
167072 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-syntax-flow/package.json'
167073 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-syntax-async-functions/package.json'
167074 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-check-es2015-constants/package.json'
167075 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-classes/package.json'
167076 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-computed-properties/package.json'
167077 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-block-scoped-functions/package.json'
167078 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-arrow-functions/package.json'
167079 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-destructuring/package.json'
167080 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-function-name/package.json'
167081 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-literals/package.json'
167082 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-block-scoping/package.json'
167083 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-for-of/package.json'
167084 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-parameters/package.json'
167085 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-object-super/package.json'
167086 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-shorthand-properties/package.json'
167087 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-spread/package.json'
167088 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-modules-commonjs/package.json'
167089 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-sticky-regex/package.json'
167090 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-template-literals/package.json'
167091 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-unicode-regex/package.json'
167092 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-flow-strip-types/package.json'
167093 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-react-display-name/package.json'
167094 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-react-jsx/package.json'
167095 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-react-jsx-source/package.json'
167096 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-template/package.json'
167097 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-es2015-typeof-symbol/package.json'
167098 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-regenerator/package.json'
167099 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-types/package.json'
167100 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-plugin-transform-strict-mode/package.json'
167101 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babel-traverse/package.json'
167102 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/buffer/package.json'
167103 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/cli-table/package.json'
167104 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/babylon/package.json'
167105 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/cardinal/package.json'
167106 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/commoner/package.json'
167107 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/cli-usage/package.json'
167108 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/cmd-shim/package.json'
167109 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/css/package.json'
167110 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/compress-commons/package.json'
167111 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/dashdash/package.json'
167112 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/decompress-zip/package.json'
167113 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/deep-eql/package.json'
167114 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/cross-spawn-async/package.json'
167115 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/crc32-stream/package.json'
167116 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/defs/package.json'
167117 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/deglob/package.json'
167118 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/detect-indent/package.json'
167119 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/electron-download/package.json'
167120 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/duplexer2/package.json'
167121 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/electron-chromedriver/package.json'
167122 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/doctrine/package.json'
167123 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/enhanced-resolve/package.json'
167124 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/esformatter/package.json'
167125 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/eslint/package.json'
167126 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/esrecurse/package.json'
167127 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/esformatter-literal-notation/package.json'
167128 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/esformatter-jsx/package.json'
167129 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/esformatter-semicolon-first/package.json'
167130 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/express/package.json'
167131 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/fresh-falafel/package.json'
167132 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/extract-zip/package.json'
167133 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/findup-sync/package.json'
167134 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/fstream/package.json'
167135 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/fsevents/package.json'
167136 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/get-package-info/package.json'
167137 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/fs-extra/package.json'
167138 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/fstream-ignore/package.json'
167139 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/glob/package.json'
167140 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/globby/package.json'
167141 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/glob-stream/package.json'
167142 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/globule/package.json'
167143 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/har-validator/package.json'
167144 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/gulp-util/package.json'
167145 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/handlebars/package.json'
167146 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/init-package-json/package.json'
167147 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/jade/package.json'
167148 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/is-number/package.json'
167149 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/inquirer/package.json'
167150 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/http-signature/package.json'
167151 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/lazystream/package.json'
167152 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/js-beautify/package.json'
167153 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/jstransform/package.json'
167154 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/js-yaml/package.json'
167155 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/liftoff/package.json'
167156 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/match-stream/package.json'
167157 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/micromatch/package.json'
167158 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/meow/package.json'
167159 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/mksnapshot/package.json'
167160 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/mv/package.json'
167161 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/node-notifier/package.json'
167162 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/npm-run/package.json'
167163 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/nugget/package.json'
167164 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/npm-registry-client/package.json'
167165 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/postcss/package.json'
167166 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/output-file-sync/package.json'
167167 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/pullstream/package.json'
167168 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/read-package-json/package.json'
167169 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/randomatic/package.json'
167170 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/read-pkg-up/package.json'
167171 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/rc/package.json'
167172 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/readline2/package.json'
167173 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/readdirp/package.json'
167174 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/recast/package.json'
167175 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/redeyed/package.json'
167176 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/regexpu/package.json'
167177 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/regenerator/package.json'
167178 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/reduce-function-call/package.json'
167179 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/request/package.json'
167180 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/rimraf/package.json'
167181 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/rocambole/package.json'
167182 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/serve-index/package.json'
167183 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/reduce-css-calc/package.json'
167184 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/rocambole-linebreak/package.json'
167185 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/send/package.json'
167186 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/source-map-support/package.json'
167187 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/serve-static/package.json'
167188 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/slice-stream/package.json'
167189 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/spawn-sync/package.json'
167190 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/standard-format/package.json'
167191 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/sockjs-client/package.json'
167192 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/standard-engine/package.json'
167193 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/tar-stream/package.json'
167194 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/svgo/package.json'
167195 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/timers-browserify/package.json'
167196 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/touch/package.json'
167197 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/temp/package.json'
167198 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/unzip/package.json'
167199 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/through2/package.json'
167200 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/uglify-js/package.json'
167201 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/stream-browserify/package.json'
167202 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/url/package.json'
167203 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/vinyl-fs/package.json'
167204 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/watchpack/package.json'
167205 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/url-parse/package.json'
167206 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/webpack-core/package.json'
167207 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/xhr/package.json'
167208 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/zip-stream/package.json'
167209 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/webpack-dev-middleware/package.json'
167210 warn enoent ENOENT: no such file or directory, open '/Users/qiaoliang89/Documents/browser-laptop/node_modules/write/package.json'
167211 verbose stack Error: abp-filter-parser-cpp@1.0.22 install: `node-gyp rebuild`
167211 verbose stack Exit status 1
167211 verbose stack     at EventEmitter.<anonymous> (/Users/qiaoliang89/.nvm/versions/node/v5.5.0/lib/node_modules/npm/lib/utils/lifecycle.js:232:16)
167211 verbose stack     at emitTwo (events.js:100:13)
167211 verbose stack     at EventEmitter.emit (events.js:185:7)
167211 verbose stack     at ChildProcess.<anonymous> (/Users/qiaoliang89/.nvm/versions/node/v5.5.0/lib/node_modules/npm/lib/utils/spawn.js:24:14)
167211 verbose stack     at emitTwo (events.js:100:13)
167211 verbose stack     at ChildProcess.emit (events.js:185:7)
167211 verbose stack     at maybeClose (internal/child_process.js:821:16)
167211 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
167212 verbose pkgid abp-filter-parser-cpp@1.0.22
167213 verbose cwd /Users/qiaoliang89/Documents/browser-laptop
167214 error Darwin 14.5.0
167215 error argv "/Users/qiaoliang89/.nvm/versions/node/v5.5.0/bin/node" "/Users/qiaoliang89/.nvm/versions/node/v5.5.0/bin/npm" "install"
167216 error node v5.5.0
167217 error npm  v3.5.3
167218 error code ELIFECYCLE
167219 error abp-filter-parser-cpp@1.0.22 install: `node-gyp rebuild`
167219 error Exit status 1
167220 error Failed at the abp-filter-parser-cpp@1.0.22 install script 'node-gyp rebuild'.
167220 error Make sure you have the latest version of node.js and npm installed.
167220 error If you do, this is most likely a problem with the abp-filter-parser-cpp package,
167220 error not with npm itself.
167220 error Tell the author that this fails on your system:
167220 error     node-gyp rebuild
167220 error You can get information on how to open an issue for this project with:
167220 error     npm bugs abp-filter-parser-cpp
167220 error Or if that isn't available, you can get their info via:
167220 error     npm owner ls abp-filter-parser-cpp
167220 error There is likely additional logging output above.
167221 verbose exit [ 1, true ]
```

**Comment by nkudo** [comment]
had same issue too, using Mac os running on Nodev5.4

**Comment by wiredearp** [comment]
Got mine on Ubuntu 14.04 using on Nodev4.2.4.

**Comment by sirugh** [comment]
Resolved updating to Node 5.5, deleting node_modules and running `npm install`.

**Comment by bbondy** [comment]
Closing this based on ^
